### PR TITLE
fix(CX-2953): adjust About Artwork section

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-2953] 

### Description

- add Rarity to the Artwork's About section
- remove "About the work" title
- remove "Purchase Details" section and move Price Paid field to the artwork details section
- add Location field to the artwork details to match the implementation on Force
- display "----" when the data is not available 

| Before | After |
| --- | --- |
| simulator_screenshot_66BABCC7-60EC-44B9-AC31-36717B229D21 | Simulator Screen Shot - iPhone SE (3rd generation) - 2022-11-18 at 14 25 07 |

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- update About section on My Collection Artwork Page -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2953]: https://artsyproduct.atlassian.net/browse/CX-2953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkAbout/MyCollectionArtworkAboutWork.tsx [hunk 1]
```diff
@@ -35,19 +47,28 @@ export const MyCollectionArtworkAboutWork: React.FC<MyCollectionArtworkAboutWork
...
 
+const MetaDataField: React.FC<{ label: string; value: string | null | undefined }> = ({
+  label,
+  value,
+}) => {
+  return (
+    <Field label={label} value={value || EMPTY_VALUE} color={!value ? "black60" : "black100"} />
+  )
+}
+
```
Comment (MrSltun) [inline src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkAbout/MyCollectionArtworkAboutWork.tsx:71]: **Suggestion:** Maybe we could move this logic to `Field.tsx` ?

**Review by MrSltun — DISMISSED** [review]
Great work Daria! 🌟 
I just left one small suggestion, but feel free to merge :)

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkAbout/MyCollectionArtworkAboutWork.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkAbout/MyCollectionArtworkAboutWork.tsx:71]: Niiice, I agree

**Review by MounirDhahri — APPROVED** [review]
🙌

**Review by MrSltun — APPROVED** [review]
💫

**Pull request merged**